samplesheets/views_api.py

```diff
@@ -421,3 +422,40 @@ def get(self, request, **kwargs):
             except Exception as ex:
                 return Response(str(ex), status=500)
         return Response(ret, status=200)
+
+
+class ProjectIrodsFileListAPIView(SODARAPIBaseProjectMixin, APIView):
+    """
+    Return a list of files in the project's sample data repository.
+
+    **URL:** ``/samplesheets/api/file/list/{Project.sodar_uuid}``
+
+    **Methods:** ``GET``
+
+    **Parameters:**
+
+    - ``project``: Project SODAR UUID
```

Comment:
We can actually remove the "parameters" part from the docstring, as the project UUID is already provided as a part of the URL itself.